ui/static/app/module/directives/stage/swStage.html.js

```diff
@@ -0,0 +1,15 @@
+angular.module("sevenWonder")
+	.directive("swStage", function () {
+		return {
+			restrict: 'EA',
+			scope: {
+				stage: '='
+			},
+			replace: true,
+			templateUrl: 'app/module/directives/stage/swStage.html',
+			controller: function ($scope) {
```

Comment:
Please, next time remove code that is not used, in this case remove controller and link